### PR TITLE
BDLaethf28

### PR DESCRIPTION
**Pull request by thea-m**
Hey everyone, this is the last manuscript from me. See also https://github.com/BetaMasaheft/Works/pull/690
The description is very minimal. A more detailed description of the content would be desirable, but would take up considerable resources. If you think that this is not up to our catalogue's standards, we will reconsider and see what can be done.

### CODE REVIEW

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Dear Dorothea, thank you.
The text has no internal subdivision, so it is difficult to produce a more detailed description of the content without going through the entire text page after page, which would be much time-consuming and - I think - would exceed the purpose of our catalogue. So, I think we can stick to the current description.
I am uncertain about  the transcription of some poorly readable portions in Amharic. Perhaps they should be revised by a native speaker. Has Solomon already seen them when he prepared the first draft of the description?

**Review by gnizla — APPROVED** [review]

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Thea! In my opinion the description of the manuscript is fine for the catalogue, even if it is not detailed as the others. The introductory note is informative.
I have just a question, ca we give the information contained in JJ handout? I think it is interesting.

**Comment by thea-m** [comment]
Can this be merged @karljonaskarlsson ?

File: OxfordBodleian/Juel-Jensen/BDLaethf28.xml [hunk 1]
```diff
@@ -133,13 +174,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                        <!-- <decoDesc>
 
                         </decoDesc>-->
 
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#38"/>
+                                    <desc type="findingAid">‘Nr. 19’ written in Amharic in the upper margin.</desc>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethf28.xml:196]: Is "Nr." a normal translation of the Amharic? If so, I think we should use "No." instead for "number". Here and some lines below.

File: OxfordBodleian/Juel-Jensen/BDLaethf28.xml [hunk 2]
```diff
@@ -133,13 +174,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
 
+                                <desc>At least two hands were involved in the writing of the manuscript, possibly more. 
+                                    The script changes repeatedly throughout, often within the same page (e.g. <locus target="#35 #90 #115"/>), 
+                                    leading to an overall impression of irregularity. All hands belong to the nineteenth–twentieth century.</desc>
+                                <date notBefore="1800" notAfter="1998"/>
```
Comment (karljonaskarlsson) [inline OxfordBodleian/Juel-Jensen/BDLaethf28.xml:180]: If we do not use the information from the handout, does it make sense to say "1998"? Maybe rather an even number?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Pull request merged**